src/binance_market.cpp

```diff
@@ -177,6 +177,41 @@ binanceError_t binance::Market::getTickSize(const char *symbol, double& maxPrice
 	return status;
 }
 
+binanceError_t binance::Market::getMinNotional(const char *symbol, double& minNotional)
```

Comment:
Excellent function! Let's make this into a separate commit?

Reply:
thanks! :)